Content.Shared/Roles/JobRequirements.cs

```diff
@@ -224,5 +235,14 @@ public static class JobRequirements
                     throw new NotImplementedException();
             }
         }
+
+        private static string ConvertTimeSpanToHoursMinutes(TimeSpan timeSpan)
+        {
+            var hours = (int) timeSpan.TotalHours;
+            var minutes = timeSpan.Minutes;
+
+            var formattedTimeLoc = Loc.GetString("role-timer-time-format", ("hours", hours), ("minutes", minutes));
+            return formattedTimeLoc;
+        }
```

Comment:
Pretty sure C# has string formatters for this.

Reply:
That part's copied from `PlayTimeStatsWindow.cs`,  I guess whoever wrote it wants the H and M to be localizable.

Reply:
Both probably should be converted to something like
```c#
return timeSpan.ToString(Loc.GetString("role-timer-time-format"));
```
Then the Loc string can use anything from [the docs](https://learn.microsoft.com/en-us/dotnet/api/system.timespan.tostring?view=net-8.0#system-timespan-tostring(system-string)).

Reply:
[4920b73](https://github.com/space-wizards/space-station-14/pull/29796/commits/4920b730ce06249463d26814dce5259899b44bbf)